nab/detectors/numenta/numenta_detector.py

```diff
@@ -66,28 +68,12 @@ def handleRecord(self, inputData):
 
 
   def initialize(self):
-    calcRange = abs(self.inputMax - self.inputMin)
-    calcPad = calcRange * .2
+    # Get config params, setting the RDSE resolution
```

Comment:
I'm not sure this strategy matches the NAB requirements. Currently you are not allowed to look ahead and use data values to set your parameters. The only exception is that we assume min/max for each metric are known to the detector (this is a reasonable assumption in most practical scenarios).  Passing in the entire dataset to some black box to set parameters however is not allowed.